app/assets/javascripts/showterm.js

```diff
@@ -75,10 +75,26 @@ $.fn.showterm = function (options) {
             window.setTimeout(tick, timings[position + 1][0] * delay);
         }
     }
-
-    $that.html('<div class="showterm-controls"><a target="_top" class="showterm-logo-link" href="/"><span class="showterm-logo">showterm</span></a><div class="showterm-slider"></div><a href="#slow">slow</a><a href="#fast">fast</a><a href="#stop">stop</a></div>');
+    help_html = "<h3> Use following keyboard shortcuts for navigation: </h2> " + 
+        "<ul>" +
+            "<li> Spacebar : Play/Pause playback</li>" +
+            "<li> shift + left/right arrow : Rewind/Forward playback" +
+            "<li> '[' and ']' : Slow down/speed up playback" +
+            "<li> ? : Show this help" +
+            "<li> escape : Close this help if open" +
+        "</ul>"
+    $that.html('<div class="showterm-controls"><a target="_top" class="showterm-logo-link" href="/"><span class="showterm-logo">showterm</span></a><div class="showterm-slider"></div><a href="#slow">slow</a><a href="#fast">fast</a><a href="#stop">stop</a><a id="help_link" href="javascript:void">Help</a></div><div id="help_dialog">' + help_html + '</div>');
```

Comment:
@prongs Can we use "?" instead of "Help" (if not, "help" should have no capital).